src/Http/Resources/ReviewResource.php

```diff
@@ -14,13 +14,6 @@ class ReviewResource extends JsonResource
      */
     public function toArray($request)
     {
-        return [
-            'id' => $this->id,
-            'rating' => $this->rating,
-            'comment' => $this->comment,
-            'reviewer' => $this->reviewer,
-            'reviewer_photo' => $this->reviewer_photo,
-            'created_at' => $this->created_at,
-        ];
+        return parent::toArray($request);
```

Comment:
I don't want to expose everything from the model to the API endpoint, just a reference number and those attributes.

Reply:
Let's use this instead of the ID:

`$table->string('google_ref')->unique();`

We also need the reviwed_at field and may do some formatting of things so that it is easier for the frontend of the website to consume this API endpoint and not make any adjustments to the data it returns.